website/src/layouts/base/header/Navigation.tsx

```diff
@@ -34,50 +35,64 @@ export function Navigation() {
-                                    <MegaMenuListEntry
-                                        key={entry.label}
-                                        label={entry.label}
-                                        externalLink={entry.externalLink}
-                                        href={entry.href}
-                                        className={entry.underlineColor}
-                                    />
-                                ))}
-                            </MegaMenuSection>
-                        ))}
-                    </MegaMenu>
+                <li>
+                    <MegaMenuNavigationEntry label='Pathogens'>
+                        <MegaMenu>
+                            {pathogenMegaMenuSections.map((section) => (
+                                <MegaMenuSection key={section.headline} {...section}>
+                                    {section.navigationEntries.map((entry) => (
+                                        <MegaMenuListEntry
+                                            key={entry.label}
+                                            label={entry.label}
+                                            externalLink={entry.externalLink}
+                                            href={entry.href}
+                                            className={entry.underlineColor}
+                                        />
+                                    ))}
+                                </MegaMenuSection>
+                            ))}
+                        </MegaMenu>
+                    </MegaMenuNavigationEntry>
                 </li>
                 <li className='h-full'>
-                    <NavigationEntry href={Page.dataSources} label='Data sources' />
+                    <SimpleNavigationEntry href={Page.dataSources}>Data sources</SimpleNavigationEntry>
                 </li>
             </ul>
         </nav>
     );
 }
 
-export type NavigationEntryProps = {
-    label: string;
-    href?: string;
+const commonNavigationEntryCss = `flex ${headerHeight} items-center text-gray-500 transition-all hover:border-b-2 hover:border-black hover:text-black`;
+
+export type SimpleNavigationEntryProps = {
+    href: string;
 };
 
-export function NavigationEntry({ href, label, className }: WithClassName<NavigationEntryProps>) {
-    const content = (
-        <div className={`flex ${headerHeight} items-center text-gray-500 transition-all hover:text-black ${className}`}>
-            {label}
-        </div>
+function SimpleNavigationEntry({ href, children }: PropsWithChildren<SimpleNavigationEntryProps>) {
+    return (
+        <a className={commonNavigationEntryCss} href={href}>
+            {children}
+        </a>
     );
+}
+
+const overlayThatClosesMenuOnClickOutside =
+    'before:cursor-default group-open:before:bg-gray-500/50 group-open:before:block group-open:before:absolute group-open:before:top-0 group-open:before:left-0 group-open:before:w-full group-open:before:h-[100vh]';
```

Comment:
I would prefer the background to be less intense. Especially, since it also shown above the menu. 
```suggestion
    'before:cursor-default group-open:before:bg-gray-500/10 group-open:before:block group-open:before:absolute group-open:before:top-0 group-open:before:left-0 group-open:before:w-full group-open:before:h-[100vh]';
```

Reply:
I tried to move the background down. So the background only covers the space below the menu. This way, one can still click on the other menu buttons (but the menu no longer closes when clicking there, but this would be fine for me)
```suggestion
    'before:cursor-default group-open:before:bg-gray-500/50 group-open:before:block group-open:before:absolute group-open:before:top-0 group-open:before:left-0 group-open:before:w-full group-open:before:h-[90vh] group-open:before:top-[10vh]';
```

Reply:
I made it `/30`. I found `/10` too transparent.